### ISSUE TITLE
bittrex feed negotiate unknown

### ISSUE THREAD

**Issue by themagicmountain** [coinbase/gdax-tt#91]
Looks like maybe the signalr client is totally not working for the bittrex feed? 

```
Error Message:  Negotiate Unknown
Exception:  undefined
Error Data:  503
``` 

is the error message I get, and 

```
➜  my-gdax-tt git:(bittrex) ✗ grep -inIEr --color=ALWAYS "Negotiate Unknown" .
./node_modules/signalr-client/signalR.js:110:                    onError('Negotiate Unknown', undefined, res.statusCode);
```

**Comment by themagicmountain** [comment]
I believe this fixes it and the earlier bug I brought up: 
https://github.com/coinbase/gdax-tt/compare/master...themagicmountain:bittrex?expand=1

**Comment by CjS77** [comment]
Fixed in #102
